framework/test/jest/unit/specs/modules/chain/bft/scenarios/11_delegates.csv

```diff
@@ -0,0 +1,75 @@
+Round,Delegate,Previous,Active Round,Height,,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,
```

Comment:
does this file need to be `CSV`?
Prefer to have in format that is more readable

Reply:
CSV is the most readable format, just open it in a right tool.